bench-vortex/src/datasets/file.rs

```diff
@@ -34,6 +34,7 @@ pub async fn convert_parquet_to_vortex(input_path: &Path, dataset: BenchmarkData
         BenchmarkDataset::ClickBench { .. } => {
             crate::clickbench::convert_parquet_to_vortex(input_path).await
         }
+        _ => todo!(),
```

Comment:
What happens here in the tpcds case?

Reply:
We never do this, instead we use duckdb to generate vortex, directly, this will be removed once I can update tpch